### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by kingkatari** [Yue-Yang/ChatGPT-Siri#98]
In ", no value was found for dictionary key 'choices'

**Comment by FulAppiOS** [comment]
Same error

**Comment by Revisor** [comment]
Try changing the model from `gpt-4` to `gpt-3.5-turbo`. It is possible your account doesn't have API access to GPT4 (yet).

**Comment by FulAppiOS** [comment]
It doesn’t work  Il giorno 1 apr 2023, alle ore 18:43, Revisor ***@***.***> ha scritto:﻿
Try changing the model from gpt-4 to gpt-3.5-turbo. It is possible your account doesn't have API access to GPT4 (yet).

**Comment by Giannicarducci** [comment]
Yeah same error... i tried with a different account and a new API - and still its giving me that error... but it was working yesterday totally fine !

**Comment by 1core2life** [comment]
In my case, I forgot to check my balance in OpenAI account. If you didn't add payment method, Would you give it a try?

**Comment by kingkatari** [comment]
Yeah I can try

**Comment by lzwjava** [comment]
<img width="654" alt="Screenshot 2023-04-03 at 23 57 54" src="https://user-images.githubusercontent.com/5022872/229563885-1ae29be0-80d4-4200-87c7-7205d8588120.png">

In my side , it is the above error.
